### PR TITLE
upgrade to bigtable 0.9.3

### PR DESCRIPTION
**Pull request by rgruener**

### CODE REVIEW

File: scio-bigtable/src/main/scala/com/spotify/scio/bigtable/package.scala [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
 package com.spotify.scio
 
+import com.google.bigtable.repackaged.com.google.cloud.grpc.BigtableClusterUtilities
```
Comment (nevillelyh) [inline scio-bigtable/src/main/scala/com/spotify/scio/bigtable/package.scala]: This is shaded. Is it intended for public use? At least I'd add a wrapper or type alias in https://github.com/spotify/scio/blob/master/scio-bigtable/src/main/scala/com/spotify/scio/bigtable/package.scala#L40

File: scio-bigtable/src/main/scala/com/spotify/scio/bigtable/package.scala [hunk 2]
```diff
@@ -80,11 +81,13 @@ package object bigtable {
       if (self.isTest) {
         // No need to update the number of nodes in a test
       } else {
-        BigtableUtil.updateNumberOfBigtableNodes(
```
Comment (nevillelyh) [inline scio-bigtable/src/main/scala/com/spotify/scio/bigtable/package.scala]: A wrapper would make this backwards compatible and reduce Java-y boilerplate.

File: scio-bigtable/src/main/scala/com/spotify/scio/bigtable/package.scala (hunk 1 above)
Comment (rgruener) [inline scio-bigtable/src/main/scala/com/spotify/scio/bigtable/package.scala]: Its in the release notes so it must be intended for public use. It probably is just shaded in the hbase-dataflow-connector for some reason https://github.com/GoogleCloudPlatform/cloud-bigtable-client/releases

File: scio-bigtable/src/main/scala/com/spotify/scio/bigtable/package.scala (hunk 2 above)
Comment (nevillelyh) [inline scio-bigtable/src/main/scala/com/spotify/scio/bigtable/package.scala]: @rgruener can you wrap `BigtableClusterUtilities` in a `object BigtableUtil` to maintain backwards compatibility?

**Comment by nevillelyh** [comment]
:+1:

**Pull request merged**